packages/router/tsconfig.json

```
@@ -0,0 +1,16 @@
+{
+	"$schema": "https://json.schemastore.org/tsconfig.json",
+	"extends": "../../tsconfig.base.json",
+	"compilerOptions": {
+		"rootDir": "src",
+		"declarationDir": "build-types",
+		"types": [ "gutenberg-env" ],
+		"checkJs": false
```

Don't allow JS in the package:

```suggestion
		"allowJs": false,
		"checkJs": false
```